tools/src/main/java/org/apache/pdfbox/tools/PDFWithTimeNewRomanText.java

```diff
@@ -21,9 +21,19 @@ public static void main(String[] args) {
             // For writing to a page content stream
```

Comment:
Some of the documentation should relate to how this helps fulfill a requirement. 
In this case the answer is that it is a mechanism for font changing that we will integrate to the text annotation class in the future.